### DISCUSSION TITLE
Temperatur Probe

### DISCUSSION THREAD

**Discussion by wanderingsocks** [BernhardSchlegel/BierBot-Bricks#35] [General]
The temp probe is plugged in but not reading during calibration or manual mode on Sonoff TH Origin TR320 20AMP.

Probe is the WTS01  

Anything i need to do get it working?

**Comment by BernhardSchlegel** [comment]
Hey there, there is a discussion going on the forum, see here: https://forum.bierbot.com/viewtopic.php?t=97 

I don't have a WTS01 myself. If someone sends over a probe I'm happy to implement it. Other than that someone needs to file a PR. 

Otherwise you have to use the DS18B20 probe.

Repo for TH Origin can be found here: https://github.com/BernhardSchlegel/Brick-32

Best,
Bernhard.
